worldengine/drawing_functions.py

```diff
@@ -159,124 +159,124 @@ def _draw_cold_parklands(pixels, x, y):
     r = 105 - b
     g = 96 - b
     b = 38 - int(b / 2)
-    pixels[x, y] = (r, g, b, 255)
+    pixels[y, x] = (r, g, b, 255)
```

Comment:
I started noticing in ```erosion.py``` that a couple of arrays were created by using ```(width, height)``` as dimensions - while all "major" arrays were created using ```(height, width)```. So I changed that and one thing lead to another...All changes in this file swap x- and y-coordinates. I ran a couple of tests to make sure the output stayed exactly the same, so there are some (~2) lines that do slightly more than just switching x and y.